src/dialogs/zha-device-info-dialog/dialog-zha-device-info.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  LitElement,
+  html,
+  css,
+  CSSResult,
+  TemplateResult,
+  customElement,
+  property,
+} from "lit-element";
+import "@polymer/paper-dialog-scrollable/paper-dialog-scrollable";
+import "../../components/dialog/ha-paper-dialog";
+import "../../panels/config/zha/zha-device-card";
+
+import { PolymerChangedEvent } from "../../polymer-types";
+import { haStyleDialog } from "../../resources/styles";
+import { HomeAssistant } from "../../types";
+import { ZHADeviceInfoDialogParams } from "./show-dialog-zha-device-info";
+import { ZHADevice, fetchZHADevice } from "../../data/zha";
+
+@customElement("dialog-zha-device-info")
+class DialogZHADeviceInfo extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() private _params?: ZHADeviceInfoDialogParams;
+  @property() private _error?: string;
+  @property() private _device!: ZHADevice;
+
+  public async showDialog(params: ZHADeviceInfoDialogParams): Promise<void> {
+    this._params = params;
+    this._device = await fetchZHADevice(this.hass, params.ieee);
+    await this.updateComplete;
+  }
+
+  protected render(): TemplateResult | void {
+    if (!this._params || !this._device) {
+      return html``;
+    }
+
+    return html`
+      <ha-paper-dialog
+        with-backdrop
+        opened
+        @opened-changed="${this._openedChanged}"
+      >
+        <h2>
+          ${this._device.user_given_name
+            ? this._device.user_given_name
+            : this._device.name}
+        </h2>
+        <paper-dialog-scrollable>
+          <zha-device-card
+            class="card"
+            .hass="${this.hass}"
+            .device="${this._device}"
+            .narrow="${false}"
+            .showHelp="${false}"
+            .showActions="${true}"
+            @zha-device-removed="${this._onDeviceRemoved}"
```

Comment:
Why listen for this event if we're not going to do anything ? I would expect we would at least close the dialog because nothing to show ?